File: jsParser/parser/Function_call_argsVisitor.js
const modelicaVisitor = require('../antlrFiles/modelicaVisitor').modelicaVisitor
const Function_call_args = require('../domain/Function_call_args')

const Function_argumentsVisitor = require('./Function_argumentsVisitor')

class Function_call_argsVisitor {
  constructor () {
    modelicaVisitor.call(this)
    return this
  }

  visitFunction_call_args (ctx) {
    let function_arguments = null

    if (ctx.function_arguments()) {
      const function_argumentsVisitor = new Function_argumentsVisitor.Function_argumentsVisitor()
      function_arguments = function_argumentsVisitor.visitFunction_arguments(ctx.function_arguments())
    }

    return new Function_call_args.Function_call_args(function_arguments)
  }
};

Function_call_argsVisitor.prototype = Object.create(modelicaVisitor.prototype)

exports.visitFunction_call_args = this.visitFunction_call_args
exports.Function_call_argsVisitor = Function_call_argsVisitor

File: java/parser/src/main/java/gov/lbl/parser/parser/Function_call_argsVisitor.java
package gov.lbl.parser.parser;

import gov.lbl.antlr4.visitor.modelicaBaseVisitor;
import gov.lbl.antlr4.visitor.modelicaParser;
import gov.lbl.parser.domain.Function_arguments;
import gov.lbl.parser.domain.Function_call_args;

public class Function_call_argsVisitor extends modelicaBaseVisitor<Function_call_args> {
    @Override
    public Function_call_args visitFunction_call_args(modelicaParser.Function_call_argsContext ctx) {
        Function_argumentsVisitor function_argumentsVisitor = new Function_argumentsVisitor();
        Function_arguments function_arguments = ctx.function_arguments() == null ? null : ctx.function_arguments().accept(function_argumentsVisitor);
        return new Function_call_args(function_arguments);
    }
}